### PR TITLE
FIX: filter and courses list rendering for different sizes gadgets

### PR DESCRIPTION
**Pull request by YuliiaMay**
I have removed logic related to JS for rendering the filters and courses list for different sizes of gadgets.

### CODE REVIEW

File: src/components/CoursesGallery/Filters/CategoryItem/CategoryItem.jsx [hunk 1]
```diff
@@ -17,42 +18,11 @@ const CategoryItem = ({ category }) => {
...
-                                    name='checked'
-                                    size={24}
-                                    color='#000000'
-                                />                                            
-                            )
-                    } 
+                {
+                    isChecked 
+                        ? <StyledIcon name='checked' />
+                        : <StyledIcon name='unchecked' />
```
Comment (maxpopovitch) [inline src/components/CoursesGallery/Filters/CategoryItem/CategoryItem.jsx]: <StyledIcon name={ isChecked ? 'checked' : 'unchecked' } />

File: src/components/CoursesGallery/Filters/FiltersListSection/FiltersListSection.jsx [hunk 2]
```diff
@@ -23,45 +22,14 @@ const FiltersListSection = ({ title, categories }) => {
...
-                                <Icon
-                                    name='minus'
-                                    size={24}
-                                    color='#000000'
-                                />                                            
-                            )
-                    }                    
+                        isOpen
+                            ? <StyledIcon name='minus' /> 
+                            : <StyledIcon name='plus' />
```
Comment (maxpopovitch) [inline src/components/CoursesGallery/Filters/FiltersListSection/FiltersListSection.jsx]: <StyledIcon name={ isOpen ? 'minus' : 'plus' } />

File: src/components/CoursesGallery/Filters/CategoryItem/CategoryItem.jsx (hunk 1 above)
Comment (YuliiaMay) [reply src/components/CoursesGallery/Filters/CategoryItem/CategoryItem.jsx]: has been fixed

File: src/components/CoursesGallery/Filters/FiltersListSection/FiltersListSection.jsx (hunk 2 above)
Comment (YuliiaMay) [reply src/components/CoursesGallery/Filters/FiltersListSection/FiltersListSection.jsx]: has been fixed

**Review by maxpopovitch — APPROVED** [review]

**Pull request merged**